rumqttd/rumqttd.toml

```diff
@@ -78,6 +78,22 @@ next_connection_delay_ms = 1
     max_inflight_count = 500
     max_inflight_size = 1024
 
+[tcp.1]
```

Comment:
`tcp` is a bit confusing here and doesn't fit in with `protocol.server` convention, maybe we should name it `dynamic` as well?

Reply:
Well I would say that tcp makes a lot more sense than `protocol.server`. Especially if you look at the other types. There WS, Prometeus, Bridge etc. My initial plan was to get rid of protocol.server but to maintain backwards compatible they are left in.

Consider the following config to replace the one that is current default:
```id = 0

[router]
id = 0
instant_ack = true
max_segment_size = 104857600
max_segment_count = 10
max_read_len = 10240
max_connections = 10010

[tcp.1]
name = "tcp-1"
listen = "0.0.0.0:1884"
sub_protocol = "all"
#sub_protocol = "v3.1.1"
#sub_protocol = "v5"
next_connection_delay_ms = 1
    [tcp.1.connections]
    connection_timeout_ms = 60000
    max_client_id_len = 256
    throttle_delay_ms = 0
    max_payload_size = 20480
    max_inflight_count = 500
    max_inflight_size = 1024
    dynamic_filters = true

[ws]
name = "ws-1"
listen = "0.0.0.0:8080"
sub_protocol = "all"
#sub_protocol = "v3.1.1"
#sub_protocol = "v5"
next_connection_delay_ms = 1
    [tcp.1.connections]
    connection_timeout_ms = 60000
    max_client_id_len = 256
    throttle_delay_ms = 0
    max_payload_size = 20480
    max_inflight_count = 500
    max_inflight_size = 1024
    dynamic_filters = true

[prometheus]
listen = "127.0.0.1:9042"
interval = 1

[console]
listen = "0.0.0.0:3030"```

This to me feels a lot more obvious about which parts does what. Maybe I am alone in thinking this? 😄

Reply:
yeah actually i just realized about the `ws` part, with that it does make more sense. 

But i am not sure whats more appropriate `tcp`, `ws`, etc... or having `v4`, `v5` etc... and specifying transport layer inside. 

will get back you after discussing with team. Marking this as resolved for now.